test/behaviour/resolution/ResolutionSteps.java

```diff
@@ -79,7 +80,14 @@ public void for_graql_query(String graqlQuery) {
 
     @Then("in reasoned keyspace, answer size is: {number}")
     public void reasoned_keyspace_answer_size_is(final int expectedCount) {
-        resolution.manuallyValidateAnswerSize(queryToTest, expectedCount);
+        final Transaction testTx = reasonedSession.transaction(Transaction.Type.READ);
```

Comment:
we should use consistent terminology now: `test` is not in use anymore - we use `reasoned` everywhere instead

Reply:
Consider it consistencised.